src/routes/create-receipt-movement.test.js

```diff
@@ -151,4 +126,48 @@ describe('Create Receipt Movement Route', () => {
     })
     expect(h.code).toHaveBeenCalledWith(500)
   })
+
+  it('should correctly pass otherReferencesForMovement to backend service', async () => {
```

Comment:
we test schema validation for otherReferencesForMovement in schema tests and happy path is already tested in a test above. I think we can remove this test as we cover all the bases already.